source/2020-02-14-the-ember-times-issue-135.md

```diff
@@ -48,13 +49,16 @@ Be sure to be on the lookout for more updates from the Ember Times team as more
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [There is no party like an Octane party 🎊](https://changelog.com/jsparty/114)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+On the 6th of February [Jen Weber @jenweber](https://github.com/jenweber) and [Chris Manson @mansona
```

Comment:
Can we add parentheses around the GitHub handles? For example, `Jen Weber (@jenweber)`.